src/main/java/org/takes/rs/RsJSON.java

```diff
@@ -89,7 +90,9 @@ public RsJSON(final Response res) {
      */
     private static byte[] print(final RsJSON.Source src) throws IOException {
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        Json.createWriter(baos).write(src.toJSON());
+        final JsonWriter writer = Json.createWriter(baos);
```

Comment:
@dmzaytsev let's replace this with a try-with-resources - `JsonWriter` is `Closeable`, so I think it might work. If not, let's close it in a `finally` block